hs_app_timeseries/forms.py

```diff
@@ -318,28 +574,97 @@ def _get_cv_dropdown_widget_items(dropdown_items, selected_item_name):
     # create a list of tuples
     cv_items = [(item.name, item.name) for item in dropdown_items]
 
-    # add the selected item as a tuple to the beginning of the list of items
-    # so that it will be displayed as the currently selected item
-    cv_items = [(selected_item_name, selected_item_name)] + cv_items
+    # alphabetize the terms
+    cv_items.sort(key=lambda tup: tup[0])
```

Comment:
wondering whether it is simpler and hence better to sort ```dropdown_items``` first before forming ```cv_items``` from it, which would not require passing a key function for sorting.